docs/guide/developing/resources/create.md

```diff
@@ -0,0 +1,110 @@
+# Create
+
+## Description
+
+Device with collection resource allows dynamic creation of resources of selected type.
+
+## Guide
+
+To develop your own device you can examine the example in [cloud_server](https://github.com/iotivity/iotivity-lite/blob/master/apps/cloud_server.c). Lets examine the example to identify the necessary steps that allow a device to dynamically create resources.
+
+1. Create a collection resource
+```C
+  oc_resource_bind_resource_type(col, "oic.wk.col");
+```
+2. Determine which resource types can populate the collection
+```C
+  oc_collection_add_supported_rt(col, "oic.r.switch.binary");
+  oc_collection_add_mandatory_rt(col, "oic.r.switch.binary");
+```
+3. Enable create operation on the collection resource
+```C
+oc_resource_bind_resource_interface(col, OC_IF_CREATE);
+```
+4. Define constructor and destructor functions for collection item instances
+```C
+oc_collections_add_rt_factory("oic.r.switch.binary", get_switch_instance, free_switch_instance);
+```
+5. Compile and link
+
+To enable create operation in iotivity-light library compile with CREATE=1 option.
+
+```make
+make cloud_server CLOUD=1 SECURE=0 CREATE=1
+```
+
+### Create a resource
+
+When you have a cloud backend and cloud_server binary running. You can use cloud client to create a resource.
+
+#### Using go grpc client
+
+Go [grpc client](https://github.com/plgd-dev/cloud/tree/master/bundle/client/grpc) is a simple tool that supports several useful commands we can combine to create a resource.
+
+1. Use the get command to get the id and href of a device with collection
+```bash
+./grpc -get
+```
+Output:
+```json
+{
+    "content": {
```

Comment:
@jkralik `GET` to collection resource shall return links. Nope ?

Reply:
it 's array of links and as i see it's empty `[]`.  grpc client doesn't decode content of resource.

Reply:
@Danielius1922 this snipped is a bit misleading. Could you please copypaste here complete snippet ? If there are too many resources, just remove them and add `...` before and after this one.